base/pop/lib/lib/getpoplog.p

```diff
@@ -3,6 +3,6 @@
 ;;;     - section independent
 ;;;     - safe against multiple reloads
 
-extend_searchlist( '$usepop/pop/getpoploglibs/auto', popautolist ) -> popautolist;
-extend_searchlist( '$usepop/pop/getpoploglibs/lib', popliblist ) -> popliblist;
-extend_searchlist( '$usepop/pop/getpoploglibs/help', vedhelplist ) -> vedhelplist;
+extend_searchlist( '$usepop/pop/getpoploglib/auto', popautolist ) -> popautolist;
```

Comment:
Was this a bug fix, or did we just not have a `getpoploglibs` folder before?

Reply:
Bug fix!